etc/config/ada.defaults.properties

```diff
@@ -4,8 +4,8 @@ defaultCompiler=gnat
 demangler=c++filt
 objdumper=objdump
 versionFlag=--version
-supportsBinary=false
-supportsExecute=false
+supportsBinary=true
+supportsExecute=true
```

Comment:
Does this mean ada now supports execution too?

Reply:
It's already the case and enabled in the amazon config. I'm simply enabling it for the default config :)